src/connector/src/parser/mod.rs

```diff
@@ -493,6 +495,147 @@ pub enum SpecificParserConfig {
     Bytes,
 }
 
+#[derive(Debug, Clone, Default)]
+pub struct ParserClientConfigList {
+    username: Option<String>,
+    password: Option<String>,
+}
+
+impl From<&HashMap<String, String>> for ParserClientConfigList {
+    fn from(props: &HashMap<String, String>) -> Self {
+        const SCHEMA_REGISTRY_USERNAME: &str = "schema.registry.username";
+        const SCHEMA_REGISTRY_PASSWORD: &str = "schema.registry.password";
+
+        ParserClientConfigList {
+            username: props.get(SCHEMA_REGISTRY_USERNAME).cloned(),
+            password: props.get(SCHEMA_REGISTRY_PASSWORD).cloned(),
+        }
+    }
+}
+
+#[derive(Default)]
+pub struct AvroProperties {
+    pub use_schema_registry: bool,
+    pub row_schema_location: String,
+    pub upsert_primary_key: String,
+    pub client_config: ParserClientConfigList,
```

Comment:
which client?

Reply:
it is used to create `Client` for schema resolver/schema registry